.gitignore

```diff
@@ -5,4 +5,5 @@ Gemfile
 Gemfile.lock
 desktop.ini
 archive/
-*.pdn
\ No newline at end of file
+*.pdn
+generated_stylesheet.css
```

Comment:
This is supposed to be generated in a tmp directory and deleted automatically, but seems its just being created at the root. I'm just gitignoring it for now